aws/request/retryer.go

```diff
@@ -29,12 +29,15 @@ var retryableCodes = map[string]struct{}{
 	"RequestError":                           {},
 	"RequestTimeout":                         {},
 	"ProvisionedThroughputExceededException": {},
-	"Throttling":                             {},
-	"ThrottlingException":                    {},
-	"RequestLimitExceeded":                   {},
-	"RequestThrottled":                       {},
-	"LimitExceededException":                 {}, // Deleting 10+ DynamoDb tables at once
-	"TooManyRequestsException":               {}, // Lambda functions
+}
+
+var throttleCodes = map[string]struct{}{
+	"Throttling":               {},
+	"ThrottlingException":      {},
+	"RequestLimitExceeded":     {},
+	"RequestThrottled":         {},
+	"LimitExceededException":   {}, // Deleting 10+ DynamoDb tables at once
+	"TooManyRequestsException": {}, // Lambda functions
```

Comment:
shouldn't `ProvisionedThroughputExceededException` be included in the `throttleCodes`